java/com/epam/ta/salad/Repository/util/VegetableFieldsValidator.java

```diff
@@ -0,0 +1,65 @@
+package com.epam.ta.salad.Repository.util;
+
+import com.epam.ta.parking.util.NullCheckUtil;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class VegetableFieldsValidator {
+
+    private final static String REG_ID_EXPRESSION = "[0-9]{1,5}";
+    private final static String REG_WEIGHT_EXPRESSION = "^-?\\d*\\.\\d+$" + "|" + "^-?\\d+$";
+    private final static String REG_NAME_EXPRESSION = "[a-zA-Z]{1,20}";
+    private final static String REG_CALORIES_EXPRESSION = "^-?\\d*\\.\\d+$" + "|" + "^-?\\d+$";
+
+    public static boolean validateId(Integer vegetableId) {
```

Comment:
when you create any entry, you doesn't have Id, why do you validate it?

Reply:
you mean remove call of this check in the add method? I was thinking that in a moment of adding we entred this data and need to validate it, doesn't it?